sycl/include/sycl/ext/oneapi/kernel_properties/properties.hpp

```diff
@@ -104,6 +104,8 @@ struct property_value<work_group_size_key, std::integral_constant<size_t, Dim0>,
   constexpr size_t operator[](int Dim) const {
     return std::array<size_t, sizeof...(Dims) + 1>{Dim0, Dims...}[Dim];
   }
+
+  constexpr size_t size() const { return sizeof...(Dims) + 1; }
```

Comment:
This is public, but is not part of the [extension](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_properties.asciidoc#kernel-properties). If we want to expose it, we need to add it to the extension too.